subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -35,6 +35,7 @@ enum flags {
 	FLAG_CTRL_SRV_MANUALLY_ENABLED,
 	FLAG_STARTED,
 	FLAG_RESUME_TIMER,
+	FLAG_AMBIENT_LUXLEVEL_SET,
```

Comment:
This flag should be added in the babblesim test main as well, to keep from stumbling on this later.